### ISSUE TITLE
Remote git using SSH

### ISSUE THREAD

**Issue by LeOSW42** [kemayo/sublime-text-git#372]
Hello !

I'm using the SFTP plugin, and that can be great to directly push the files on my server from SublimeText.
Is that possible to add an feature, to run git through SSH, using a config file to know the login, server, port, password and directory ?

Thanks for the answer, even if I suppose that's impossible

**Comment by ArKam** [comment]
@PeterPablo Are you saying that using Sublime-text-git to push local master branch to a remote origin one is already available ?? I can't find the commit and push option on STGit.

**Comment by PeterPablo** [comment]
Yes. That is exactly what I do, for example while contributing to GitHub. Do you see the other `Git` commands in the command palette of Sublime Text?

**Comment by ArKam** [comment]
Yep, I got it, I just find the push command and found the STGit for ST3 commands.
Wonderful!!

**Comment by espellcaste** [comment]
Humm, @PeterPablo I gave a pull from the plugin and it haven't sent to the repo on Bitbucket, although everything it is okay when using Git batch. Are you saying it works with SSH without making any change to the settings?

**Comment by PeterPablo** [comment]
@espellcaste, yes this works for me like a charm without fiddling with any settings. Please try to push changes from a terminal window. If this works fine without manual interaction than git is setup properly and `sublime-text-git` should consequently work as well.

**Comment by espellcaste** [comment]
That's the problem, when pushing from the terminal, it works correctly, but when from inside Sublime, it doesn't. :(
